### PR TITLE
Asemota33/homepage

### PR DESCRIPTION
**Pull request by Asemota33**
- Moved Navigation to separate page
- Started working on homepage

### CODE REVIEW

File: src/app/scoreboard/scoreboard.component.html [hunk 1]
```diff
@@ -1 +1,27 @@
-<p>scoreboard works!</p>
+<div class="homepageContainer">
```
Comment (itimie) [inline src/app/scoreboard/scoreboard.component.html:1]:
```suggestion
<div class="homepageContainer">
  <div class="homePageBG">
    <div style="" class="clr-row clr-justify-content-center">
      <h1 style="margin-top: 2rem">Where Do You Rank</h1>
    </div>

    <div style="margin-top: 2rem;" class="clr-row clr-justify-content-center">
        <div class="results_table">
            <clr-datagrid>
                <clr-dg-column style="width: 20vw">ID</clr-dg-column>
                <clr-dg-column style="width: 20vw">First Name</clr-dg-column>
                <clr-dg-column style="width: 20vw">Last Name</clr-dg-column>
                <clr-dg-column style="width: 20vw">Email</clr-dg-column>
        
                <clr-dg-row *ngFor="let user of users">
                  <clr-dg-cell>{{ user.id }}</clr-dg-cell>
                  <clr-dg-cell>{{ user.name }}</clr-dg-cell>
                  <clr-dg-cell>{{ user.creation  }}</clr-dg-cell>
                  <clr-dg-cell>{{ user.color }}</clr-dg-cell>
                </clr-dg-row>
        
                <clr-dg-footer></clr-dg-footer>
              </clr-datagrid>
        </div>
    </div>
  </div>
</div>

```

**Comment by itimie** [comment]
Please add this in styles.scss

`
.datagrid-column-title, .datagrid-cell{
  font-size:30px !important
}
`

**Comment by itimie** [comment]
Once you add all the suggestions it should look like this:

Screenshot 2024-07-10 at 2 46 54 AM

**Comment by Asemota33** [comment]
> Please add this in styles.scss
> 
> `.datagrid-column-title, .datagrid-cell{ font-size: 1.5rem !important }`

Hey Esther, for this particular one I wanted the Title to be overlapping. I was trying to recreate something like this for the table but just don't have the same font 
Screenshot 2024-07-10 205920

Currently it looks like this 
Screenshot 2024-07-10 210241

**Pull request merged**